submarine-security/spark-security/pom.xml

```diff
@@ -572,6 +572,20 @@
   </build>
 
   <profiles>
+    <profile>
+      <id>spark-3.2</id>
+      <properties>
+        <spark.version>3.2.0</spark.version>
+        <scala.version>2.12.14</scala.version>
+        <scala.binary.version>2.12</scala.binary.version>
+        <!--<scalatest.version>3.2.0</scalatest.version>-->
+        <spark.compatible.version>3</spark.compatible.version>
+        <commons-lang3.version>3.10</commons-lang3.version>
+        <jackson-databind.version>2.10.5</jackson-databind.version>
+        <jackson-annotations.version>2.10.5</jackson-annotations.version>
```

Comment:
Can we also add following here please?
`<ranger.version>2.1.0</ranger.version>`
`<ranger.major.version>2.1</ranger.major.version>`
`<httpcomponents.httpclient.version>4.5.13</httpcomponents.httpclient.version>`
`<httpcomponents.httpcore.version>4.4.15</httpcomponents.httpcore.version>`
`<httpcomponents.httpmime.version>4.5.13</httpcomponents.httpmime.version>`

Reply:
i agree @anumee , but ranger versions do have a seperate profile altogether.. as this spark 3.2 profile can be used with a different version of ranger as well.